webapp/stores/team_store.jsx

```diff
@@ -148,6 +148,21 @@ class TeamStoreClass extends EventEmitter {
     getTeamListings() {
         return this.teamListings;
     }
+
+    isTeamAdminForCurrentTeam() {
```

Comment:
There may be an another admin in that list that is not actually the current user I think, I would leave it just to be sure